app/safedigit/safedigit.ts

```diff
@@ -0,0 +1,88 @@
+import type { NewTenant } from "./types/NewTenant";
+import type { Webhooks } from "./types/Webhoook";
+import { DeliveryMethod, type WebhookHandler } from "@shopify/shopify-api";
+import type { WebhookConfig } from "node_modules/@shopify/shopify-app-remix/build/ts/server/config-types";
+
+export class SafeDigit {
+  public static baseUrl = "https://api.test.safedigit.io/public/api";
+
+  async createTenant(
+    clientId: string,
+    shop: string,
+    accessToken: string | undefined,
+  ) {
+    if (!accessToken) {
+      throw new Error("Access token is required");
+    }
+    try {
+      const newTenant: NewTenant = {
+        tenantId: SafeDigit.getTenantId(clientId, shop),
+        name: shop,
+        configuration: {
+          shopifyToken: accessToken,
+          shopifyShopDomain: shop,
+        },
+      };
+      return await fetch(`${SafeDigit.baseUrl}/${clientId}/create-tenant`, {
```

Comment:
which HTTP client use shopify?

Reply:
No idea, but I didn't want to add packages, and this one worked straight out of the box.

Reply:
I guess shopify uses axios, can you check it?